### PR TITLE
chore: remove refs to deprecated io/ioutil

### PR DESCRIPTION
**Pull request by testwill**

### CODE REVIEW

File: interp/handler.go [hunk 1]
```diff
@@ -318,7 +318,19 @@ type ReadDirHandlerFunc func(ctx context.Context, path string) ([]os.FileInfo, e
 // It makes use of [ioutil.ReadDir].
```
Comment (mvdan) [inline interp/handler.go:318]: fix this reference as well

**Review by mvdan — APPROVED** [review]

**Review by mvdan — APPROVED** [review]

**Comment by mvdan** [comment]
Hm, this appears to break the tests. I guess in some edge cases the behavior is subtly different. I honestly don't think it's worth spending the time to figure out why; per the TODO, I will swap out the API in v4 to use `fs.DirEntry` directly. Thanks again for the PR.
